x-pack/plugins/integration_assistant/common/api/model/common_attributes.ts

```diff
@@ -31,7 +31,7 @@ export const RawSamples = z.array(z.string());
  * mapping object to ECS Mapping Request.
  */
 export type Mapping = z.infer<typeof Mapping>;
-export const Mapping = z.object({}).passthrough();
+export const Mapping = z.object({});
```

Comment:
Whys is the `passThrough` removed?

Reply:
Great catch!

Reply:
It was generated like this by `node scripts/generate_openapi --rootDir ./x-pack/plugins/integration_assistant`, but I reverted it

Reply:
Yeah , the `passthrough` is needed as I understand